app/static/css/main.css

```diff
@@ -122,3 +122,27 @@ textarea {
         position: static;
     }
 }
+
+.scrollTopBtn {
+    border: none;
+    border-radius: 50%;
+    color: white;
+    background-color: #00356b;
+    cursor: pointer;
+    font-size: 30px;
+    text-align: center;
+    line-height: 38px;
+    padding-bottom: 15px;
+    width: 60px;
+    position: fixed;
+    bottom: 20px;
+    right: 20px;
+    z-index: 100;
+    opacity: 0;
+    transform: translateY(100px);
```

Comment:
It strikes me that you could use the `bottom` property to achieve this animation without having to use a transform